docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.profile

```diff
@@ -343,9 +344,11 @@ function su_humsci_profile_contextual_links_alter(array &$links, $group, array $
       $link['title'] .= " {$entity_types[$group]}";
     }
   }
+
   if (
-    !in_array($group, ['media', 'block_content']) &&
-    !\Drupal::currentUser()->hasPermission('view all contextual links')
+    !in_array($group, ['media', 'block_content', 'hs_blocks.social_media_block']) &&
+    !$current_user->hasPermission('view all contextual links') &&
+    !$current_user->hasPermission('edit social media block')
```

Comment:
This is allowing any user with the "edit social media block" to see any contexual menu, including types that were previously exclude like paragraphs. It's necessary to ensure that only the block types in the list have contextual links for any user without the "view all contextual links" permission.

Reply:
Removed this code and added in our new custom group `social_media_block` with seems to work fine now.